services/authentication/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@bloom-housing/authentication-service",
+  "version": "0.0.1",
+  "description": "Authentication service for Bloom affordable housing system",
+  "main": "dist/index.js",
+  "author": "Ben Peters <bencpeters@gmail.com>",
+  "license": "GPL-3.0",
+  "private": true,
+  "dependencies": {
+    "@types/node": "^13.13.4",
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "helmet": "^3.22.0",
+    "oidc-provider": "^6.24.0",
+    "rimraf": "^3.0.2",
+    "typescript": "^3.8.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.9.6",
+    "@babel/preset-typescript": "^7.9.0",
+    "@types/jest": "^25.2.1",
+    "babel-jest": "^25.5.1",
+    "dotenv": "^8.2.0",
+    "jest": "^25.5.2",
+    "ts-node-dev": "^1.0.0-pre.44"
+  },
+  "engines": {
+    "node": "12",
+    "yarn": "^1.22"
+  },
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "yarn run clean && tsc -p tsconfig.prod.json",
+    "start": "yarn install && yarn run build && node dist/src/index.js",
+    "dev": "NODE_ENV=development ts-node-dev src/index.ts",
+    "test": "jest"
+  },
+  "babel": {
```

Comment:
Just to mostly contradict myself on the record about babel versioning above, I think we're mainly using it for testing, and I would be thrilled to take it out of the process entirely in favor of ts-jest running TS more natively. I made a couple of attempts with limited luck and I don't want to make it a requirement to move forward with this, but maybe starting a new service is the right time to give it another shot?

Reply:
I can take a look!